packages/collaboration/src/userinfopanel.tsx

```diff
@@ -41,27 +58,79 @@ export class UserInfoPanel extends Panel {
 /**
  * A SettingsWidget for the user.
  */
-export class UserInfoBody extends ReactWidget {
-  private _user: User.IIdentity;
-
+export class UserInfoBody
+  extends ReactWidget
+  implements Dialog.IBodyWidget<User.IManager>
+{
+  private _userManager: User.IManager;
+  private _translation: IRenderMime.TranslationBundle;
   /**
    * Constructs a new settings widget.
    */
-  constructor(user: User.IIdentity) {
+  constructor(props: UserInfoProps) {
     super();
-    this._user = user;
+    this._userManager = props.userManager;
+    this._translation = props.translation;
```

Comment:
This needs to be named `_trans`:

```suggestion
    this._trans = props.translation;
```

Because it gets extracted using very simple static analysis, see https://jupyterlab.readthedocs.io/en/latest/extension/internationalization.html#rules

Reply:
Do you mean renaming also the property name in the options ?

Reply:
> but I thought the name was required when using it only.

Yes, that's correct. My bad I misread the extend of changes in your previous commit.

Reply:
:smile: took me 5 min reading again, to be sure I was not missing something again.